rebar.config

```diff
@@ -18,11 +18,13 @@
     {layout, edoc_layout_chunks},
     {preprocess, true},
     {dir, "_build/default/lib/hex_core/doc"}]}.
+{deps, [
+    {gpb, "4.17.6"}
```

Comment:
could you double check it actually works, eg set this to 4.17.5, rm hex_pb_x and ensure that they were generated with the correct version.

Reply:
Hm... it worked bumping forward, now I'm getting some sort of ```ERROR: To build outside a git work tree, you must explicitly
           set the version, either by writing it to a file gpb.vsn,
           or by creating a versioned archive, and then unpack and
           build that.``` stuff

Reply:
I was not doing `as dev` 😄. Should we put `gpb` inside that profile? It's the one that'll be used to re-generated the files, any way. Unless you prefer it to not be everytime the lib. is compiled.

Reply:
> could you double check it actually works, eg temporarily set this to 4.17.5, rm hex_pb_x and ensure that they were generated with the correct version.

4.17.5 (🎯) was a buggy version 😢 But I did a couple of commits for you to check how "changing gpb would work" (in between I had to delete the _pb_x files, sure - not sure we couldn't automate this with a rebar3 command or alias).